### PR TITLE
Fix a few typos

### PR DESCRIPTION
**Pull request by Blake-Madden**

### CODE REVIEW

File: docs/extensibility/visualstudio.extensibility/inside-the-sdk/remote-ui.md [hunk 1]
```diff
@@ -378,7 +378,7 @@ Update `MyToolWindowContent.xaml` to use the new properties in the data context.
 
 ![Diagram of tool window with two-way binding and a command.](./media/remote-ui-commands.gif)
 
-## Understanding asynchronicity in Remote UI
+## Understanding Asynchronicity in Remote UI
```
Comment (Court72) [inline docs/extensibility/visualstudio.extensibility/inside-the-sdk/remote-ui.md:381]:
```suggestion
## Understanding asynchronicity in Remote UI
```

File: docs/msbuild/msbuild-targets.md [hunk 2]
```diff
@@ -237,7 +237,7 @@ Get the paths for the .NET Framework installation directory
 These paths are not used directly by this .targets file but are available for pre and
 post build steps.
 
-This is a generally overriden target, for example it is overriden in the Microsoft.NETFramework.targets file
+This is a generally overridden target, for example it is overriden in the Microsoft.NETFramework.targets file
```
Comment (Court72) [inline docs/msbuild/msbuild-targets.md:240]:
```suggestion
This is a generally overridden target. For example, it's overridden in the Microsoft.NETFramework.targets file.
```

**Comment by Court72** [comment]
@ghogen, @mijacobs

Can you review the proposed changes in [docs/msbuild/msbuild-targets.md](https://github.com/MicrosoftDocs/visualstudio-docs/pull/10182/files#diff-27128ad4d28b700e70457dd5b36da4737d84f24356516d74ed3956ff227a11e1)? 

If this content is copied directly from *Microsoft.Common.CurrentVersion.Targets*, can you update the source to correct the typos?

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by Mikejo5000 — APPROVED** [review]

**Review by ghogen — APPROVED** [review]

**Review by mgoertz-msft — APPROVED** [review]

**Comment by mgoertz-msft** [comment]
#signoff for the Modeling changes

**Pull request merged**